### DISCUSSION TITLE
A blog: "Web 1-two-Web 3: Convenience unlocked🔓"

### DISCUSSION THREAD

**Discussion by prajjwal-mathur** [EddieHubCommunity/support#3646] [Show and tell]
# Hey everyone👋🏽
### So, after researching a while for these terms(around 2-3 days) on LFX and Binance Academy site, I made a Hashnode Blog, so that we all can have a quick read.
- Check out the blog: [Web 1-two-Web 3: Convenience unlocked🔓](https://slowflash.hashnode.dev/web-1-two-web-3-convenience-unlocked)
- *Make sure to leave your precious feedback or any advice about the article.*

**Comment by meenal21** [comment]
😄 Quick Read and informative! 

You can also add in a summarized table for the key differences between the three versions.

**Reply by prajjwal-mathur** [reply]
Thank you @meenal21 for such kind words, I'll make the necessary changes 🫡
